### DISCUSSION TITLE
How to constrain renovate to a particular publication channel?

### DISCUSSION THREAD

**Discussion by jgoux** [renovatebot/renovate#8601] [Request Help] [answered]
Hello all,

We setup renovate in our repo and it's awesome!

One little issue we have right now is that renovate is updating our `parcel` dependency to nightly builds, even if the original dep was installed from the `next` channel : 

![image](https://user-images.githubusercontent.com/1443499/107340016-91a7f980-6abd-11eb-8d6c-893c18d6f5a0.png)

We'd like to stay on the `next` channel and don't have renovate to upgrade to nightly. How can we do this?

Thanks for your help!

**Comment by viceice** [comment, marked as answer]
checkout https://docs.renovatebot.com/configuration-options/#followtag

**Reply by jgoux** [reply]
Thanks for the very quick answer! 👍 
My issue with followTag is that when 2.0.0 will hit latest, I'll have to update renovate's config right?
Is there any other solution that I can use to have a more future proof setting?

**Reply by rarkins** [reply]
You could use regex patterns with `allowedVersions`, to exclude nightly releases, e.g. something like:

```json
{
  "packageRules": [{
    "packageNames": ["parcel"],
    "allowedVersions": "!/nightly/"
  }]
}
```

That way the rule has no point or effect once 2.0.0 becomes `latest`.

**Reply by jgoux** [reply]
This is perfect, thanks both of you! What a *blazing fast* support. ⚡ 😄
